lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -514,9 +514,8 @@ def _deploy_environment(self, environment: Environment, log: Logger) -> None:
             normalized_name = constants.NORMALIZE_PATTERN.sub("-", constants.RUN_NAME)
             # Take last chars to make sure the length is to exceed max 90 chars
             # allowed in resource group name.
-            resource_group_name = truncate_keep_prefix(
-                f"{normalized_name}-e{environment.id}", 80
-            )
+            rg_name = f"{normalized_name}-e{environment.id}"
```

Comment:
This part should be ok, the resource group name is generated, and it should start with "lisa-". The original logic can double check it.

Reply:
Yes, will revert this line change.